### DISCUSSION TITLE
Does triggering events count towards visit duration?

### DISCUSSION THREAD

**Discussion by jonaspiela** [plausible/analytics#1463] [Anything Else]
Hey!

Visitors of our [site](https://pielaco.com/podcast) have the ability to play podcasts straight from the website using a html player. So people stay on the site, listen to audio often times longer than 30min, and don't do nothing. This should be counted towards a single session and towards their visit duration. In Google Analytics we would trigger "listening" events every 10 seconds to keep the session "alive".

In Plausible,
1. I am not sure if events count towards duration since [the support page](https://plausible.io/docs/metrics-definitions#visit-duration) doesn't explicitly state that and
2. this method is spamming the system with countless events that count towards our page views limit (not good).

So my questions are
1. do events count towards visit duration and
2. is there another way to keep the session alive without sending countless events?

**Comment by metmarkosaric** [comment]
Hey @jonaspiela! All events are considered interactive with Plausible so they count towards the bounce rate and visit duration calculations. In the future, we plan to move to the beacon API (#190) to be able to determine a much more accurate visit duration without the need for any manual action.

**Reply by jonaspiela** [reply]
Thanks for your feedback. Looking forward to that beacon implementation! Meanwhile, we stick with "listening pings" to track the visit duration without any need for the user to navigate pages.
